scripts/eurostat/birth_death_migration/EurostatNUTS3_BirthDeathMigration_statvar.mcf

```diff
@@ -0,0 +1,43 @@
+Node: dcid:Count_Person
+typeOf: dcs:StatisticalVariable
+populationType: schema:Person
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+
+Node: dcid:Count_BirthEvent
+typeOf: dcs:StatisticalVariable
+populationType: dcs:BirthEvent
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+
+Node: dcid:Count_MortalityEvent
+typeOf: dcs:StatisticalVariable
+populationType: schema:MortalityEvent
+measuredProperty: dcs:count
+statType: dcs:measuredValue
```

Comment:
Could you remove these variables since they already exist? We recently solved the inconvenience of having to redefine existing variables, so you actually don't need to do this at all anymore.